README.md

```diff
@@ -302,6 +303,20 @@ Combining this with the Spring Boot Auto Configuration can make integration very
 
 ## Known Issues
 
+### Age Verification
+
+#### Affects
+
+* Versions prior to 1.4.2
+
+#### Description
+
+A bug in the isAgeVerified() property of the profile meant that isAgeVerified() would always return null (i.e. not checked).
```

Comment:
Could you please format the method name like this `isAgeVerified()`?

Reply:
Sure